src/routes/Home.tsx

```diff
@@ -0,0 +1,5 @@
+export const Home = () => {
+  return (
```

Comment:
return statement is redundant here, you can use just brackets here as you did in `PageNotFound.tsx`